src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -144,8 +144,8 @@ internal static object GetPrimitiveDefaultValue(Type type)
                     return default(float);
                 case TypeCode.Double:
                     return default(double);
-                //case TypeCode.DBNull:
-                //    return default(DBNull);
+                case TypeCode.DBNull:
+                    return default(DBNull);
```

Comment:
Yeah, that helper method was put in place when porting the code from .NET Framework to .NET Core initially. We can revert back now.